src/main/java/seedu/address/logic/commands/CreateFolderCommand.java

```diff
@@ -21,6 +21,8 @@
 
     public static final String MESSAGE_SUCCESS = "New folder added: %1$s";
     public static final String MESSAGE_DUPLICATE_FOLDER = "This folder already exists in UNIon";
+    public static final String MESSAGE_FOLDER_NAME_TOO_LONG = "This folder name is too long!"
+            + "Please keep it to a maximum of 30 chars";
```

Comment:
can you add a space in between the two strings?
![image](https://user-images.githubusercontent.com/30801910/140248646-0de65088-493b-4c7b-8178-138f9afa2b17.png)